config/environments/development.rb

```diff
@@ -8,10 +8,10 @@
   # In the development environment your application's code is reloaded any time
   # it changes. This slows down response time but is perfect for development
   # since you don't have to restart the web server when you make code changes.
-  config.cache_classes = false
+  config.cache_classes = true
 
   # Do not eager load code on boot.
-  config.eager_load = false
+  config.eager_load = true
```

Comment:
Are we sure about that?

Reply:
the comment!! 😢

Reply:
Oops. No we're not. That slipped through from my testing to ensure that we don't have an autoload issue.